dep/reflex-platform/github.json

```diff
@@ -1,8 +1,8 @@
 {
   "owner": "reflex-frp",
   "repo": "reflex-platform",
-  "branch": "master",
+  "branch": "android-allow-post-build-signing",
```

Comment:
Not sure it makes sense to depend on a branch of reflex-platform;  lets get this merged there, too.